packages/nimble-components/src/table/specs/table-column-specs/table-column-anchor-hld.md

```diff
@@ -84,9 +80,9 @@ Arrowing to a anchor table cell should focus the link (if it is actually a link
 
 In the accessibility tree, the cells of an anchor column are instances of [`nimble-table-cell`](https://github.com/ni/nimble/blob/f663c38741e731bef91aa58e8fb2d1cec653b679/packages/nimble-components/src/table/components/cell/template.ts#L6) which has a `role` of [`cell`](https://w3c.github.io/aria/#cell). The cell then has a child `nimble-anchor`, which has a `role` of [`link`](https://w3c.github.io/aria/#link).
 
-### Angular RouterLink Support
+### Angular Routing support
```

Comment:
@atmgrifter00 @fredvisser @rajsite Where did we end up on this? I remember us changing our minds about the need to implement this column type. We should update this PR and the related work item.